microservices-connector/internal/controller/gmconnector_controller.go

```diff
@@ -297,7 +302,7 @@ type TeiEmbeddingCfg struct {
 	HttpsProxy       string
 }
 
-type RetriverCfg struct {
+type RetrieverCfg struct {
```

Comment:
Is it possible to remove all these structs?  We can apply what user defined in config directly:

          config:
            endpoint: /v1/chat/completions
            hostPath: /root/GMC/data/tgi
            modelId: ise-uiuc/Magicoder-S-DS-6.7B

Reply:
Hi @irisdingbj, `endpoint` should be leaved for http verb between microservices. `hostPath` can be removed, and `modelId` should be provided by oneclick's configmap(They may create a configmap yaml if not existing).